open-banking-accelerator/components/com.wso2.openbanking.accelerator.identity/src/main/java/com/wso2/openbanking/accelerator/identity/token/wrapper/ResponseWrapper.java

```diff
@@ -81,4 +81,3 @@ public void setContentType(String type) {
         super.setContentType(type);
     }
 }
-
```

Comment:
is this change is intentional?

Reply:
Yes there were few extra lines at the end of this class.